kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/LoggerFacade.kt

```diff
@@ -0,0 +1,30 @@
+package kotlinx.fuzz
+
+import kotlin.reflect.full.primaryConstructor
+
+@Suppress("OBJECT_NAME_INCORRECT")
+object KLoggerFactory {
+    const val LOGGER_IMPLEMENTATION_PROPERTY = "kotlinx.fuzz.logger.implementation"
+
+    fun getLogger(clazz: Class<*>): KLogger {
```

Comment:
How about to write an overload for `KClass<*>`. This way, we wont need to write `.java` when calling it.

`fun getLogger(clazz: KClass<*>): KLogger = getLogger(clazz.java)`

Reply:
Fixed